yb-voyager/cmd/importDataFileCommand.go

```diff
@@ -326,6 +327,19 @@ func setDefaultForNullString() {
 	}
 }
 
+func setDefaultForDelimiter() {
+	if delimiter == "" {
```

Comment:
Nit: Not much of a problem for this specific function. But in idiomatic go, we try to reduce indentation by inverting the checks and returning early e.g. this function can be implemented as:
```
func setDefaultForDelimiter() {
  if delimiter != "" {
    return
  }
  switch fileFormat {
    ...
  }
}
```
Think as if you pay Rs 10 for the first level of indentation, Rs 100 for the second level, 1000 for the next, and so on!! Now the objective is to save as much money as you can ;-)

Reply:
Yes, got it. Thanks for the example :)